core/rawdb/database_quorum.go

```diff
@@ -81,28 +86,39 @@ func GetPrivateBlockBloom(db ethdb.Database, number uint64) (bloom types.Bloom)
 	return bloom
 }
 
-type PrivacyMetadataLinker interface {
-	PrivacyMetadataRootForPrivateStateRoot(privateStateRoot common.Hash) common.Hash
-	LinkPrivacyMetadataRootToPrivateStateRoot(privateStateRoot, privacyMetadataRoot common.Hash) error
+// AccountExtraDataLinker maintains mapping between root hash of the state trie
+// and root hash of state.AccountExtraData trie
+type AccountExtraDataLinker interface {
```

Comment:
QUESTION: why to we need to create a `AccountExtraDataLinker` interface? I am asking because for accessing the private state root we did not need that. Thanks

Reply:
You are right, we don't access private state root with the linker. It only helps to maintain the mapping for AccountExtraData trie